edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookFrameworkServiceImpl.java

```diff
@@ -118,7 +120,9 @@ public void addGradebook(final String uid, final String name) {
             // Check for null default.
             if (defaultGradeMapping == null) {
                 defaultGradeMapping = (GradeMapping)gradeMappings.iterator().next();
-                if (log.isWarnEnabled()) log.warn("No default GradeMapping found for new Gradebook=" + gradebook.getUid() + "; will set default to " + defaultGradeMapping.getName());
+                if (log.isWarnEnabled()) {
+					log.warn("No default GradeMapping found for new Gradebook=" + gradebook.getUid() + "; will set default to " + defaultGradeMapping.getName());
```

Comment:
Should use newer slf4j logging methods:
`log.warn("No default GradeMapping found for new Gradebook={}; will set default to {}", gradebook.getUid(), defaultGradeMapping.getName());`